### ISSUE TITLE
Responsible person for the webpage

### ISSUE THREAD

**Issue by jdittrich** [wmde/software.wikimedia.de#49]
**Story:** I would like to change something on this webpage and I don't know who is the responsible person who has the final say. ( @johl , @samu-wmde , is it you?)

**Comment by JeroenDeDauw** [comment]
I created the site (mainly to aid us in recruitment efforts) and sort of have ownership of the project. However everyone is free to edit as they see fit (I wrote this to the list at some point). The content used to be on MediaWiki.org, and while this site is not a wiki, there is little reason to be more restrictive about its content now beyond limiting merges to people with merge access (everyone in the WMDE org). If there is some kind of disagreement that cannot be resolved by the participants then ultimately, in my understanding, the authority lies with the head of department.

Perhaps we should mention this somewhere. How about the README file in this repo?

**Comment by JeroenDeDauw** [comment]
TL;DR: If you want to make a change, just make a PR, and have someone merge it, or merge it yourself if you think the review is not worth it for that change.

**Comment by jdittrich** [comment]
I'm fine, @samu-wmde, can you close this issue if the readme is OK for you?

**Comment by JeroenDeDauw** [comment]
Samu approved
